doc/nrf/test_and_optimize.rst

```diff
@@ -71,7 +71,7 @@ Use one of the following methods:
 
    .. group-tab:: nRF Connect for Visual Studio Code
```

Comment:
In other group-tabs you have nRF Connect for VS Code. Maybe just for consistency, have it like that also here.

Reply:
Won't look into shortcut, as it is not really part of this PR.